ckanapi/cli/load.py

```diff
@@ -249,3 +264,37 @@ def _copy_from_existing_for_update(obj, existing, thing):
     if thing in ('organizations', 'groups'):
         if 'users' not in obj and 'users' in existing:
             obj['users'] = existing['users']
+
+def _upload_resources(ckan,obj,arguments):
+    resources = obj['resources']
+    if len(resources)==0:
+        return
+    for resource in resources:
+        if resource['url_type'] == 'upload':      # check for same domain resources
+            for key in resource.keys():
+                if isinstance(resource[key],(dict,list)):
+                    resource.pop(key)                # dict/list objects can't be encoded
+            if arguments['--upload-resources']:
+                f = requests.get(resource['url'],stream=True)
+                new_url = resource['url'].rsplit('/',1)[-1]
+                resource['upload'] = (new_url,f.raw)
+            else:
+                resource['url_type'] = ''           # hack url_type so that url can be modified
+                resource['package_id'] = obj['name']
+            ckan.action.resource_update(**resource)
+
+
+def _upload_logo(ckan,obj):
+    for key in obj.keys():
+        if isinstance(obj[key],(dict,list)):
+            obj.pop(key)                            #dict/list objects can't be encoded
+    if 'http' in obj['image_url']:                  # logo is an external link
```

Comment:
obj['image_url'].startswith('http') would be a little better. Even better would be to check for a netloc returned from urlparse https://docs.python.org/2/library/urlparse.html